locales/pl.json

```diff
@@ -310,207 +410,90 @@
-    "search_filters_date_option_year": "W tym roku",
-    "search_filters_type_option_video": "Wideo",
+    "search_filters_date_option_week": "Ten tydzień",
+    "search_filters_date_option_month": "Ten miesiąc",
+    "search_filters_date_option_year": "Ten rok",
+    "search_filters_type_label": "Typ",
+    "search_filters_type_option_all": "Dowolny typ",
+    "search_filters_type_option_video": "Film",
     "search_filters_type_option_channel": "Kanał",
     "search_filters_type_option_playlist": "Playlista",
-    "search_filters_type_option_movie": "Film",
-    "search_filters_type_option_show": "Pokaż",
-    "search_filters_features_option_hd": "HD",
-    "search_filters_features_option_subtitles": "Napisy/CC",
-    "search_filters_features_option_c_commons": "Creative Commons",
-    "search_filters_features_option_three_d": "3D",
+    "search_filters_type_option_movie": "Film kinowy",
+    "search_filters_type_option_show": "Program telewizyjny",
+    "search_filters_duration_label": "Długość",
+    "search_filters_duration_option_none": "Dowolna długość",
+    "search_filters_duration_option_short": "Poniżej 4 minut",
+    "search_filters_duration_option_medium": "Od 4 do 20 minut",
+    "search_filters_duration_option_long": "Powyżej 20 minut",
+    "search_filters_features_label": "Cechy",
     "search_filters_features_option_live": "Na żywo",
     "search_filters_features_option_four_k": "4K",
-    "search_filters_features_option_location": "Lokalizacja",
+    "search_filters_features_option_hd": "HD",
+    "search_filters_features_option_subtitles": "Napisy",
+    "search_filters_features_option_c_commons": "Licencja Creative Commons",
+    "search_filters_features_option_three_sixty": "360°",
+    "search_filters_features_option_vr180": "VR180",
+    "search_filters_features_option_three_d": "3D",
     "search_filters_features_option_hdr": "HDR",
-    "Current version: ": "Aktualna wersja: ",
+    "search_filters_features_option_location": "Lokalizacja",
+    "search_filters_features_option_purchased": "Kupiony",
+    "search_filters_sort_label": "Sortuj według",
+    "search_filters_sort_option_relevance": "Trafność",
+    "search_filters_sort_option_rating": "Ocena",
+    "search_filters_sort_option_date": "Data przesłania",
+    "search_filters_sort_option_views": "Liczba wyświetleń",
+    "search_filters_apply_button": "Zastosuj wybrane filtry",
+    "Current version: ": "Bieżąca wersja: ",
     "next_steps_error_message": "Po czym należy spróbować: ",
-    "next_steps_error_message_refresh": "Odśwież",
-    "next_steps_error_message_go_to_youtube": "Przejdź do YouTube",
-    "invidious": "Invidious",
-    "tokens_count_0": "{{count}} token",
-    "tokens_count_1": "{{count}} tokeny",
-    "tokens_count_2": "{{count}} tokenów",
-    "generic_videos_count_0": "{{count}} film",
-    "generic_videos_count_1": "{{count}} filmy",
-    "generic_videos_count_2": "{{count}} filmów",
```

Comment:
Do note that because polish has three plural forms (e.g: 1 miesiąc / 2 miesiące / 5 miesięcy, [as per the unicode standard](https://www.unicode.org/cldr/charts/42/supplemental/language_plural_rules.html#pl)) you need to keep the suffixes `_0`, `_1` and `_2`. If you use no suffix / `_plural`, then the strings won't be translated.